libs/pageserver_api/src/models.rs

```diff
@@ -1496,6 +1502,9 @@ pub struct TimelineInfo {
 
     /// The status of the rel_size migration.
     pub rel_size_migration: Option<RelSizeMigration>,
+
+    /// Whether the timeline is invisible in synthetic size calculations.
```

Comment:
when is it `None`?

Reply:
The Option is added because we need to handle backward compatibility (deploy pageserver first -> storcon cannot parse the request)